### PR TITLE
Fix/code cleanup

### PR DESCRIPTION
**Pull request by Nirma**
This is just a bunch of minor fixes that remove unnecessary things like trailing commas, use of semicolons and whitespace.

### CODE REVIEW

**Comment by krzyzanowskim** [comment]
did you use `swiftformat` for that?

**Comment by Nirma** [comment]
These were issues the popped during a run of `swiftlint`, I know the project prefers `swiftformat` but these minor issues I thought were general enough to warrant a fix. What do you think? Sorry if its too insignificant.

File: Tests/CryptoSwiftTests/XCTestManifests.swift [hunk 1]
```diff
@@ -13,7 +13,7 @@ extension AESCCMTests {
         ("testAESCCMTestCase3", testAESCCMTestCase3),
         ("testAESCCMTestCase3Decrypt", testAESCCMTestCase3Decrypt),
         ("testAESCCMTestCase3DecryptPartial", testAESCCMTestCase3DecryptPartial),
-        ("testAESCCMTestDVPT256", testAESCCMTestDVPT256),
```
Comment (krzyzanowskim) [inline Tests/CryptoSwiftTests/XCTestManifests.swift]: It will back as this is how it's autogenerated by `swift test --generate-linuxmain`

**Comment by krzyzanowskim** [comment]
@Nirma while on that, mind to run swiftformat as well? it touches many files so we can do it all at once

**Comment by Nirma** [comment]
@krzyzanowskim I just gave swiftformat and run and pushed, it should be all caught up now! 
How is it?

**Comment by krzyzanowskim** [comment]
I just realized I didn't put `.swiftformat` for the project. I'm sorry about that. I'll do it in this PR and run swiftformat again

**Comment by krzyzanowskim** [comment]
uh, I can't push to your branch (I forgot). I commit `.swiftformat` to `master` branch. You should rebase and use it here.

**Comment by Nirma** [comment]
@krzyzanowskim Ok I just rebased with master and gave swiftformat another run through.

Hows it look now?

**Comment by krzyzanowskim** [comment]
@Nirma almost there, however https://github.com/krzyzanowskim/CryptoSwift/pull/759/commits/4388eb26633bba90a355cf163e93c3b9c30578cd introduced changes that I'd rather avoid. If you can remove this commit please

**Comment by Nirma** [comment]
@krzyzanowskim Ok! I just took that commit out, how about now?
Sorry for all the back and fourth.

**Comment by Nirma** [comment]
@krzyzanowskim Ok, should be no conflicts now. What do ya think?

**Comment by krzyzanowskim** [comment]
looks good thank you!

Screenshot 2019-10-22 at 09 58 35

**Comment by Nirma** [comment]
Awesome! Thanks for letting me contribute! Yeah 105 files is a bit for one PR haha.

**Pull request merged**